src/main/java/com/faforever/client/vault/search/SearchController.java

```diff
@@ -126,7 +126,14 @@ public void setSearchableProperties(Map<String, String> searchableProperties) {
   public void setSortConfig(ObjectProperty<SortConfig> sortConfigObjectProperty) {
     sortPropertyComboBox.getItems().addAll(searchableProperties.values());
     sortOrderChoiceBox.getSelectionModel().select(sortConfigObjectProperty.get().getSortOrder());
-    sortPropertyComboBox.getSelectionModel().select(searchableProperties.get(sortConfigObjectProperty.get().getSortProperty()));
+
+    String savedSortProperty = searchableProperties.get(sortConfigObjectProperty.get().getSortProperty());
+    
+    if (savedSortProperty == null)
+        savedSortProperty = searchableProperties.values().iterator().next();
```

Comment:
Please enable automatic source code formatting in the commit dialog.

Reply:
Also, there's no need to explicitly select something. The following code would've been simpler and not abuse the variable:

```suggestion
    if (savedSortProperty != null) {
      sortPropertyComboBox.getSelectionModel().select(savedSortProperty);
    }
```